### DISCUSSION TITLE
Translations to Spanish

### DISCUSSION THREAD

**Discussion by SofiaGC009** [microsoft/AI-For-Beginners#380] [Translations]
Hi everyone,

I’d like to contribute by translating the README and other important texts into **Spanish** for the repo. I believe this will help make the content more accessible to Spanish-speaking users. 🌟

Is it okay if I proceed with it?

Thanks!

**Comment by leestott** [comment]
Hi Translation are now automatic using https://github.com/Azure/co-op-translator
